examples/kernel32.rs

```diff
@@ -0,0 +1,117 @@
+#![allow(non_upper_case_globals)]
+
```

Comment:
You use `strlen` further down, so just make sure to import it up top.

```suggestion
use libc::strlen;
```